index.html

```diff
@@ -8,31 +9,227 @@
     <link rel="stylesheet" href="styles.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
+        rel="stylesheet">
 </head>
+
 <body>
-    <div class="wrapper">
-        <header>
-            <div id="name"><a href="#">John Doe</a></div>
-            <nav class="toolbar">
-                <a id="hamburger" href="#"><img src="icons/menu.svg"></a>   
-            </nav>
-        </header>
-        <main>
-            <section class="headline-section">
-                    <h1>Hey There.<br>I’m Alex</h1>
-                    <h4>I’am a Software Developer</h4>
-                    <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.</p>
-                    <div class="socials">
-                        <a class="social" href=""><img src="icons/github.svg" /></a>
-                        <a class="social" href=""><img src="icons/linkedin.svg" /></a>
-                        <a class="social" href=""><img src="icons/angelist.svg" /></a>
-                        <a class="social" href=""><img src="icons/twitter.svg" /></a>
-                        <a class="social" href=""><img src="icons/medium.svg" /></a>
-                    </div>
-            </section>
-        </main>      
-    </div>
+    <header>
+        <div id="name"><a href="#">John Doe</a></div>
+        <nav class="toolbar">
+            <a id="hamburger" href="#"><img src="icons/menu.svg"></a>
+        </nav>
+    </header>
+    <main>
+        <section class="headline-section">
+            <h1>Hey There.<br>I’m Alex</h1>
+            <h4>I’am a Software Developer</h4>
+            <p id="description">I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="socials">
+                <a class="social" href=""><img src="icons/github.svg" /></a>
+                <a class="social" href=""><img src="icons/linkedin.svg" /></a>
+                <a class="social" href=""><img src="icons/angelist.svg" /></a>
+                <a class="social" href=""><img src="icons/twitter.svg" /></a>
+                <a class="social" href=""><img src="icons/medium.svg" /></a>
+            </div>
```

Comment:
- [ ] [**OPTIONAL**] Kindly use `ul` and `li` to list these social media icons :+1: , It will make our ode more semantic :100:

Reply:
> * [x]  [**OPTIONAL**] Kindly use `ul` and `li` to list these social media icons +1 , It will make our ode more semantic 100
 It must be mandatory not optional remove optional tag

Reply:
> > * [x]   [**OPTIONAL**] Kindly use `ul` and `li` to list these social media icons +1 , It will make our ode more semantic 100
> >   It must be mandatory not optional remove optional tag

Alredy fixed, pushed and requested for review @MaganaAsiati